src/common/net.spec.ts

```diff
@@ -21,22 +21,14 @@ describe("Net", () => {
 
     context("when the port is available", () => {
       it("should return a fulfilled promise", () => {
-        expect(isPortAvailable(port, host)).to.eventually.be.fulfilled;
+        expect(isPortAvailable(port + 1, host)).to.eventually.be.fulfilled;
```

Comment:
I’m not sure I followed the +1 here. Is 4568 available, but 4567 isn’t? Is that specified soemwhere that I missed?

Reply:
I wasn't able to fully track down the sequence of events, but it seems as if the OS doesn't fully release the port in the check when the next assertion is running - and thus the port isn't available.

...and then I realised my mistake - not _returning_ the `expect`. 🤦‍♂️

Fixed.